bin/run.sh

```diff
@@ -25,19 +23,16 @@ function main {
     return 0;
   fi
 
-  # JUnit results contain the unit test failures only. But they contain `@testdox` - readable test names.
-  # Teamcity results contain user output in addition to failures as "failed risky tests"
-  #   (command line arguments --disallow-test-output and --fail-on-risky).
-  # At the moment we require both logs to provide all information to the website.
-  output=$("${PHPUNIT_BIN}" \
-    -d memory_limit=300M \
-    --log-junit "${output_dir%/}/${JUNIT_RESULTS}" \
-    --log-teamcity "${output_dir%/}/${TEAMCITY_RESULTS}" \
-    --no-configuration \
-    --do-not-cache-result \
-    --disallow-test-output \
-    --fail-on-risky \
-    "${test_files%%*( )}" 2>&1)
+  # Our PHPUnit extension writes directly to ${EXERCISM_RESULT_FILE}
+  # Our PHPUnit extension requires ${EXERCISM_EXERCISE_DIR} before PHPUnit provides it
+  output=$( \
+    EXERCISM_RESULT_FILE="${output_dir%/}/${EXERCISM_RESULTS}" \
+    EXERCISM_EXERCISE_DIR="${solution_dir%/}" \
+    "${PHPUNIT_BIN}" \
+      -d memory_limit=300M \
```

Comment:
The runners have 3G, should we allow more than 300M?

Reply:
Not sure why to raise that limit? Tests + student code are not expected to use more memory now?